components/logWorkout/SetDetails.jsx

```diff
@@ -118,10 +118,10 @@ const SetDetails = (props) => {
       </SetNumber>
 
       <PrevWeight>
-        <StyledText>{prevWeight !== 'n/a' ? `${prevWeight} lbs.` : prevWeight}</StyledText>
+        <StyledText>{prevPerRep !== 'n/a' ? `${prevPerRep} lbs.` : prevPerRep}</StyledText>
```

Comment:
I'm confused as to the structure of this because I think there is a discrepancy somewhere. I thought it would make more sense to replace the reps with the time instead of the weight in case they wanted to have weighted timed exercises. If not, I think the 'previous' has the wrong unit because based on the current structure I think it should say 60 secs not 60 lbs. Let me know if I'm confusing myself or if what I'm saying makes sense

Reply:
wait yeah you're right gdi hahaha

Reply:
good catch